functions/fisher.fish

```diff
@@ -72,13 +72,13 @@ function fisher --argument-names cmd --description "A plugin manager for Fish"
             set --local fetch_plugins $update_plugins $install_plugins
             echo (set_color --bold)fisher $cmd version $fisher_version(set_color normal)
 
-            for plugin in $fetch_plugins
+            fish_path=(status fish-path) for plugin in $fetch_plugins
                 set --local source (command mktemp -d)
                 set --append source_plugins $source
 
                 command mkdir -p $source/{completions,conf.d,functions}
 
-                fish --command "
+                $fish_path --command "
```

Comment:
Is this also Fish 3.0 or was it introduced more recently?

Reply:
I know this has been around for a while now. Just checking.

Reply:
@faho Sorry to bother but would you happen to know when variable-in-command-position was released? Tried combing through the release notes but couldn't find it.

Reply:
It seems that this is in since 3.0 as well! 🎉

Reply:
https://github.com/fish-shell/fish-shell/commit/865a4647aecbd7bf0fe55cb26f176a6646d4aed0